### ISSUE TITLE
Import/Export Grooves/FX via SysEx.

### ISSUE THREAD

**Issue by bangcorrupt** [bangcorrupt/hacktribe#30]
Add feature to import and export groove templates and FX presets via SysEx.

**Comment by gradientledge** [comment]
so is there a way to transfer gate arp patterns as well its sucks they have no triplets or just useable patterns.

**Comment by bangcorrupt** [comment]
I haven't looked for the arp patterns yet.  If I can find them, we can change them.

**Comment by gradientledge** [comment]
so for the sysex are you just using midiox to communicate with it, or is there a better program as that ones ancient?

**Comment by bangcorrupt** [comment]
> are you just using midiox to communicate with it

Answered in #37.

**Comment by bangcorrupt** [comment]
Get/Set IFX added in b700ac9.

**Comment by bangcorrupt** [comment]
Get/Set grooves added in 070b282.

**Comment by DepreTux** [comment]
> Get/Set grooves added in [070b282](https://github.com/bangcorrupt/hacktribe/commit/070b282123f87812977e1ede9c8bda226a4fb07d).

How does the "add ifx" option differ from the "set ifx" one?
Does it mean we can add IFX to the list?

Thanks!

**Comment by bangcorrupt** [comment]
Yes, `add_ifx` adds a new IFX preset to the top of the list, increasing the total count.

**Comment by DepreTux** [comment]
> 

Amazing... I'll give it a go as soon as I can figure it out!
Thanks!
